### ISSUE TITLE
Unable to load GIF images in some cases

### ISSUE THREAD

**Issue by derf** [derf/feh#222]
Just for the record: depending on the order in the filelist, gif images may be rejected as unloadable even though they are perfectly fine. This appears to be a bug introduced in imlib2 1.4.7. More details on the [Debian bug tracker](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=812657)

**Comment by Raziel-23** [comment]
I replay to your issue, because I had a similar problem, and probably I found a cause of this issue. I wasn't able to trigger this by using feh or mentioned on the mailing list imgif program, but it constantly happened to me when I was using sxiv. So I dug deeper and I found this:
https://sourceforge.net/p/giflib/mailman/message/34772182/

Now I can't reproduce this issue anymore after applying this patch from trunk:
https://sourceforge.net/p/giflib/code/ci/ef0cb9b4be572262b49fbc26fb2348683f44a517/

I just want let you know about this, because your issue was very valuable for me and led me to this patch.
